packages/athena/libs/fabric_ca_services/fabric_utils.js

```diff
@@ -55,19 +55,24 @@ module.exports = (logger, ev, t) => {
 	//------------------------------------------------------------------
 	// Generate authorization token required for accessing fabric-ca APIs
 	//------------------------------------------------------------------
-	exports.generateAuthToken = (reqBody) => {
+	exports.generateAuthToken = (reqBody, path, method) => {
 		// specific signing procedure is according to:
 		// https://github.com/hyperledger/fabric-ca/blob/master/util/util.go#L213
 		let cert = Buffer.from(t.signingIdentity.certificate).toString('base64');
-		let bodyAndCert;
+		let signString;
 		if (reqBody) {
 			let body = Buffer.from(JSON.stringify(reqBody)).toString('base64');
-			bodyAndCert = body + '.' + cert;
+			signString = body + '.' + cert;
 		} else {
-			bodyAndCert = '.' + cert;
+			signString = '.' + cert;
 		}
 
-		let sig = t.signing_lib.sign(bodyAndCert, { hashFunction: t.cryptoSuite.hash.bind(t.cryptoSuite), signingIdentity: t.signingIdentity });
+		if (path && method) {
+			const s = Buffer.from(path).toString('base64');
+			signString = method + '.' + s + '.' + signString;
```

Comment:
this line is the real/main change right? the "new method" is to sign over a string that has the method and path. make sense! 

so seems like once we merge this there will be some ancient CA versions that will no longer work with the latest console. CA's that are too old for the new way... for the sake of traceability could you jot down in this PR what min CA version the console needs now?

Reply:
Yes, this is the main change, it is consistent with how all the other Fabric CA clients were updated back in 2019.

I've updated the Description with the Fabric CA versions.